src/components/fraction-keypad.js

```diff
@@ -78,95 +88,116 @@ class FractionKeypad extends React.Component {
             dismissOrJumpOutKey = KeyConfigs.DISMISS;
         }
 
-        return <Keypad>
```

Comment:
Note: Since it does look like the expression keypad and the fraction keypad share code, I tried to create a new "keypad-segments" file that contains the separate components to be reused. However, it turns out that there are a bunch of little differences that make this less practical than it sounds. (e.g. styling on the 7 button, % button in place of cdot, minus button in place of variable key)

I decided to leave it and just wait for the new redesign to start development first.